docs/install/virtual_machine.md

```diff
@@ -175,10 +178,6 @@ plt.title('How fast do you want to go today?')
 show(plt)
 ``` 
 
+### R Extras
```

Comment:
Which pre-processor do we use for markdown (prior to Jekyll)?
I could dig around, but figured you might be able to point it out quickly.